File: testdata/bench/javascript/BenchmarkTest00011.js
// Vulnerable: SQL injection via Prisma $queryRawUnsafe
const express = require('express');
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient();
const router = express.Router();
router.get('/test011', async (req, res) => {
    const email = req.query.email;
    const users = await prisma.$queryRawUnsafe("SELECT * FROM users WHERE email = '" + email + "'");
    res.json(users);
});
module.exports = router;

File: testdata/bench/rust/BenchmarkTest00011.rs
use actix_web::{web, HttpResponse};

// Vulnerable: SQL injection via match pattern
async fn handler(path: web::Path<String>) -> HttpResponse {
    let category = path.into_inner();
    let sql = match category.as_str() {
        _ => format!("SELECT * FROM items WHERE category = '{}'", category),
    };
    // sqlx::query(&sql).execute(&pool).await
    HttpResponse::Ok().body(sql)
}
